tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+"""Pytest fixtures."""
+
+from enum import Enum
+from typing import cast
+
+import pytest
+
+from array_api_extra._lib._compat import array_namespace
+from array_api_extra._lib._compat import device as get_device
+from array_api_extra._lib._typing import Device, ModuleType
+
+
+class Library(Enum):
+    """All array libraries explicitly tested by array-api-extra."""
+
+    ARRAY_API_STRICT = "array_api_strict"
+    NUMPY = "numpy"
+    NUMPY_READONLY = "numpy_readonly"
```

Comment:
This runs numpy twice on all tests, which is less than ideal, but I could not find a cleaner generalized design.